ArduPlane/mode_autoland.cpp

```diff
@@ -175,7 +175,7 @@ bool ModeAutoLand::_enter()
     entry_alt = plane.current_loc.alt;
     
     // start first leg toward the base leg loiter to alt point
-    if (climb_first) {
+    if (climb_first && plane.control_mode_reason != ModeReason::BATTERY_FAILSAFE) {
```

Comment:
I'm not sure about this. I would rather run of of battery on the way back than skip the climb and run into the ground. Better to keep the behavior consistent to avoid confusion.

Reply:
not sure I agree on this